crates/av/src/microphone.rs

```diff
@@ -1,19 +1,12 @@
 use bevy::prelude::*;
```

Comment:
i think we could move the microphone.rs code into livekit_native.rs, so it's parallel with the web version (we can do in a separate pr if you prefer)